osu.Game/Overlays/Comments/VotePill.cs

```diff
@@ -43,6 +43,7 @@ public class VotePill : LoadingButton, IHasAccentColour
 
         private readonly BindableBool isVoted = new BindableBool();
         private readonly BindableInt votesCount = new BindableInt();
+        private bool disabled;
```

Comment:
I don't like that this exists. I understand why, but this at least needs to be negated (because now you have `Enabled` in the base and `disabled` in the derived), renamed (maybe something like `canVote`), or even removed (you should be able to just set `Enabled` to false in the ctor - if it's going to start disabled, it won't ever get to the loading part, so it shouldn't interfere - this might warrant leaving a comment in source, though).